app/jobs/locate_groups_job.rb

```diff
@@ -0,0 +1,27 @@
+class LocateGroupsJob < ActiveJob::Base
+  def perform(last_active)
+    group_ids = Discussion.select('DISTINCT(group_id) group_id, last_activity_at').
+                 where('last_activity_at > ?', last_active).
+                 order('last_activity_at desc').map(&:group_id)
+
+
+    Group.where(id: group_ids).each do |group|
```

Comment:
Am I right in sniffing that this is the most common country/region/city amongst members of a particular group? ❔